app/models/rules/basic.rb

```diff
@@ -1,4 +1,4 @@
-class Rules::Basic < Struct.new(:auction)
+class Rules::Basic < Rules::BaseRules
```

Comment:
Yeah maybe @adelevie has a good idea. I just feel like Base and Basic are easily confused.